### ISSUE TITLE
club id

### ISSUE THREAD

**Issue by bartneck** [bartneck/swiML#68]
assert that id in club index is unique

**Comment by calloc262** [comment]
xsd can't be used to check across files, would need a master file with all clubs and id's to ensure that they are unique. This would also help for lookup when creating or modifying index files for clubs.

**Comment by bartneck** [comment]
This is not what the ID is for. A single club could have multiple programs per day. Or even multiple programs per session (e.g. one for the fast swimmers and one for the slower). One group could swim in the outside pool and one in the indoor pool.

The ID is there to ensure that multiple programs per day have a unique ID.

**Comment by calloc262** [comment]
To clarify: Each Id should be unique within the given date. 
Currently the programs are listed under a session tag -> should this be changed to program given each instance refers to a separate program?

**Comment by bartneck** [comment]
You are right. The ```id``` is to identify different programs in the same day. It does not specify a club. If a club has several sessions per day, then each needs to have a unique ```id```. The ```id``` tag is indeed under the ```session``` tag.
